app/jenkins-for-jira-ui/src/App.tsx

```diff
@@ -49,17 +52,26 @@ export const environmentSettings = {
 
 const AppContainer = styled.div`
 	color: #172B4D;
-	margin: 24px 24px 24px 0;
+	margin: 24px 36px 24px 0;
 `;
 
 const App: React.FC = () => {
 	const [history, setHistory] = useState<any>(null);
+	const [renovateConfigFlag, setRenovateConfigFlag] = useState<boolean>(false);
+
+	const fetchFeatureFlag = useCallback(async () => {
+		const renovatedJenkinsFeatureFlag = await fetchFeatureFlagFromBackend(
```

Comment:
We've got a useFeatureFlag hook in the frontend which utilises launchdarkly-react-client-sdk. However, when I was updating the FF services for the frontend and the backend to be able to pass the cloudId, it doesn't' appear there's a way to retrieve target data with the target data. I want to dig into this more but, in the meantime, retrieving it via the backend will suffice. Further investigation is #tomorrowsproblem 😉